docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -109,53 +109,68 @@ and transaction family specific. When implementing a transaction handler the
 binary serialization protocol is up to the implementer.
 {% endif %}
 
-Without yet getting into the details of how this information is encoded, we can
-start to think about what ``apply`` needs to do. ``apply`` needs to
+To separate details of state encoding and payload handling from validation
+logic, the Xo example has XoState and XoPayload classes. The XoPayload has
+name, action, and space fields, while the XoState contains information about
+the game name, board, state, and which players are playing in the game.
 
-1) unpack the command data from the transaction,
-2) retrieve the game data from the context,
-3) play the game, and
-4) save the updated game data.
-
-Accordingly, a top-down approach to ``apply`` might look like this:
+The actions one can enact on a game are to create the game, take a new board space,
```

Comment:
I suggest changing this to: "Valid actions are: create a new game, take an unoccupied board space,"